pkg/api/v1/resource/helpers.go

```diff
@@ -75,8 +75,8 @@ func GetResourceRequest(pod *v1.Pod, resource v1.ResourceName) int64 {
 		return 1
 	}
 	totalResources := int64(0)
-	for _, container := range pod.Spec.Containers {
-		if rQuantity, ok := container.Resources.Requests[resource]; ok {
+	for _, workload := range pod.Spec.Workloads() {
```

Comment:
Why is this change here? Master does not have this changes

Reply:
this is a but in master where the container type is directly used instead of the workloads()_ funciton. this function is only used in the preemption cases where VM type is not there yet so this bug is not uncovered yet. this change should fix it.

Reply:
workloads() returns containers or VM depending on the type of pod, so the change looks correct.